Modules/ConfigOptions.lua

```diff
@@ -624,8 +624,11 @@ return {
 	{ var = "conditionOnConsecratedGround", type = "check", label = "Are you on Consecrated Ground?", tooltip = "In addition to allowing any 'while on Consecrated Ground' modifiers to apply,\nConsecrated Ground grants 6% Life Regeneration to players and allies.", apply = function(val, modList, enemyModList)
 		modList:NewMod("Condition:OnConsecratedGround", "FLAG", true, "Config", { type = "Condition", var = "Combat" })
 	end },
-	{ var = "conditionOnFungalGround", type = "check", label = "Are you on Fungal Ground?", ifCond = "OnFungalGround", tooltip = "Allies on your Fungal Ground gain 10% of Non-Chaos Damage as extra Chaos Damage.", apply = function(val, modList, enemyModList)
-		modList:NewMod("Condition:OnFungalGround", "FLAG", true, "Config", { type = "Condition", var = "Combat" })
+	{ var = "conditionOnFungalGround", type = "check", label = "Are you on Fungal Ground?", ifCond = "CreateFungalGround", tooltip = "Allies on your Fungal Ground gain 10% of Non-Chaos Damage as extra Chaos Damage.", apply = function(val, modList, enemyModList)
```

Comment:
There is no support for multiple conditions for config options. If I split it, the minion one is going to be visible even while the are no minions present in the build. It currently says "Allies on your Fungal Ground..."
None of the suggested mods locations seem right. Fungal ground can be created by Sporeguard when stationary so it can't be exclusive to the Contaminate skill. I don't think it's correct to put them in mod parser just because currently all sources come from one item. It could work like consecrated ground which only uses config to set a flag and the regen mod is added in `CalcSetup`.
The way the flags are set there is no issue with "ghost" mods. There is a "ghost" checkbox instead. Try this:
1. Equip Sporeguard - checkbox appears
2. Enable Fungal Ground - mod applies
3. Un-equip Sporeguard - mod doesn't apply, checkbox stays
4. Disable Fungal Ground - checkbox disappears

Reply:
I think it would be great if you find a solution that doesn't create a "ghost checkbox". I'd even prefer it to generate a "ghost mod" instead, so that when we're able to fix the "ghost" issue, we only have to fix one bug, not two.

Reply:
There is no solution without issues. If you don't want any trouble with config, then the mods have to stay in mod parser. If we ever get another source of Fungal Ground, this problem will return.